tools/make-dtrace.sh

```diff
@@ -44,6 +45,7 @@ tar cvf ../../out/crucible-dtrace.tar \
     sled_upstairs_info.d \
     trace-vol.d \
     tracegw.d \
+    up-info.d \
```

Comment:
Do you also want to deploy the new `simple.d`?

Reply:
I'm planning on polishing it a bit more, so no, not yet :)